Management/src/prometheus/prometheus.yml

```diff
@@ -0,0 +1,25 @@
+global:
+  scrape_interval: 1s # makes for a good demo
+
+scrape_configs:
+  - job_name: 'ActuatorWeb'
+    basic_auth:
+      username: actuatorUser
+      password: actuatorPassword
+    static_configs:
+      - targets: ['host.docker.internal:8090']
+    metrics_path: /actuator/prometheus
+    # The configuration below skips certificate validation
+    # scheme: https
+    # tls_config:
+    #   insecure_skip_verify: true
```

Comment:
This doesn't actually work when uncommented. Prometheus fails with:
```
Error scraping target: Get "https://host.docker.internal:8091/actuator/prometheus": http: server gave HTTP response to HTTPS client
```
We should either get it to work and include this in non-commented-out form, or leave it out altogether.

Reply:
No, it does not. Delete all containers and volumes, start from scratch, and you'll get the error above.